internal/core/src/index/VectorMemIndex.cpp

```diff
@@ -953,6 +954,14 @@ VectorMemIndex<T>::LoadFromFileV2(const Config& config) {
                strerror(errno));
     LOG_INFO("load vector index done");
 }
+
+template <typename T>
+ResourceUsage
+VectorMemIndex<T>::GetResourceUsage() const {
+    // TODO: we can't get the memory usage of vector index now.
+    return resource_usage_;
```

Comment:
if mmap is not enbaled, resource usage should be all memory usage. but I dind't see any calculation here.

Reply:
Ok, I will use file size as mem size if disable mmap